README.rst

```diff
@@ -0,0 +1,26 @@
+Tests
+=====
+
+Clear Containers Tests Repository
+
+Running functional tests
+------------------------
+
+To run functional tests::
+
+  $ make functional
+
+`Environment variables`_ can be used to override the paths of Clear Containers components.
+For example::
+
+  $ RUNTIME="cc-runtime" make functional
+
+in the above example the installed version of the Runtime is used
+
+
+Environment variables
+---------------------
+
+- `RUNTIME` - Path of Clear Containers Runtime, the default path is ``$GOPATH/src/github.com/clearcontainers/runtime/cc-runtime``
```

Comment:
Ah no sorry, the default values are from the Makefile, not the default values of the arguments. Confusing.